### PR TITLE
Translation documentation

### PR DESCRIPTION
**Pull request by andyinabox**
### What
Add documentation for translation feature to address #106 and #131 

### Why
Make it easier to deal with translations

### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [ ] Tapping on a marker on the map displays information about the marker in a popup.
- [ ] Tapping the "Show list of locations" button replaces the map view with a list view.
- [ ] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [ ] Tapping one of the ✅'s in the legend filters items on the map and in the list.
- [ ] Changing the language to spanish changes things on the page.
- [ ] Clicking the Help/Info button opens and closes a menu with information.
- [ ] Clicking the X Close button on the Help/Info screen closes the modal.

### Check the app in the following web browsers:
- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari

### CODE REVIEW

File: docs/LANGUAGE_TRANSLATION.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+Here's the basic format of the spreadsheet:
+
+| id | eng | spa | 
+| :--------- | :-------- |:--------- |
+| lang_name | English | Español |
+| lang_name_in_eng | English | Spanish |
+| language | Language | Idioma |
+| welcome | Welcome! | ¡Bienvenidos! |
+
+The first column (e.g. `lang_name`) and first row (e.g. `eng`) are **required** and must be filled out for any added row/column. The rest of the cells can be filled in as new translated terms are available.
```
Comment (experimatt) [inline docs/LANGUAGE_TRANSLATION.md:48]: I think you mean "The first two rows (language key & lang_name) are required when adding a new column."

**Comment by experimatt** [comment]
@andyinabox ok here's some quick PR specific feedback:

- As a whole I think it looks good and we should merge this so there's a place people can refer to for information about how we're doing translations. Nice work on documenting all this!
- We should change the sheet name in the spreadsheet from "Sheet1" to something more descriptive. I noticed this the other day when adding a new sheet for translator contact info, but didn't want to rename it at the time bc I didn't want to break anything.

**Big picture feedback (cc @jdalt)**

High level, this documentation is good, but _long_. It goes to show that this is complicated. Based on the process as we've outlined it, I wonder if we should switch to a more traditional i18n implementation, with language specific json files where we've saved the hard-coded translations for these words/phrases straight in the repo. Considering we're not giving translators direct access to the spreadsheet anyway, saving the translations statically in a different format would just be a change of format.

This is all just off the cuff gut reactions, so feel free to ignore. But as it exists now it's very complicated, and as we continue adding new functionality/features and keep changing how we're communicating things on the site, we're going to continue to need translation help (regardless of how this is implemented, I guess).

File: docs/LANGUAGE_TRANSLATION.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+A good resource for hard-to-find flag images:
+
+https://en.wikipedia.org/wiki/Unrepresented_Nations_and_Peoples_Organization#Members
+
+A checklist for adding new flag images
+
+ - [ ] The image should be a PNG
+ - [ ] The image should be 23px wide
+ - [ ] The image name should have the format `lang-XXX.png` where "XXX" is the 3-letter language code
+ - [ ] The image should be placed in the `/public/images` directory in this repo
```
Comment (caseyhelbling) [inline docs/LANGUAGE_TRANSLATION.md:111]: Per the conversation in slack. What to do if there isn't a flag... Maybe fallback to an SVG of the 3 letter 639-2 iso code?

Comment (jdalt) [reply docs/LANGUAGE_TRANSLATION.md:111]: I like the idea of a 3 letter fallback.

**Comment by jdalt** [comment]
> High level, this documentation is good, but long. It goes to show that this is complicated. Based on the process as we've outlined it, I wonder if we should switch to a more traditional i18n implementation, with language specific json files where we've saved the hard-coded translations for these words/phrases straight in the repo. Considering we're not giving translators direct access to the spreadsheet anyway, saving the translations statically in a different format would just be a change of format.

> This is all just off the cuff gut reactions, so feel free to ignore. But as it exists now it's very complicated, and as we continue adding new functionality/features and keep changing how we're communicating things on the site, we're going to continue to need translation help (regardless of how this is implemented, I guess).

If the the google sheet doesn't facilitate easier access by the translators then I agree that going to an i18n json file would be a better solution. We gain back version control and a pull request workflow. It's one less external link to manage. We might also want to consider other i18n SaaS solutions, but the alternatives might lose on complexity.

**Comment by caseyhelbling** [comment]
I’ve been thinking about this all afternoon — I agree we should go to a version controlled file (I’ve also been thinking about what the next step is as it relates to the spreadsheet as a backend ... but that’s a separate conversation.)

**Comment by andyinabox** [comment]
@experimatt @jdalt @caseyhelbling 

:+1: Agree on moving to JSON! I thought the Google sheet might help get us up and running sooner (not sure if that turned out to be the case), but putting something in the repo definitely makes more sense in the long run. We should probably also add the flag image path to the configuration so we have more control over that.

Does anyone know if good tools/services for i18n that we might want to move toward? On my initial many of them seemed like overkill, but not sure I'll feel the same now :)

**Comment by experimatt** [comment]
New issue created for moving translations from google sheets to something version controlled: https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/issues/145

**Pull request merged**